packages/cozy-client/src/models/file.js

```diff
@@ -47,11 +47,13 @@ export const isFile = file => file && file.type === FILE_TYPE
 export const isDirectory = file => file && file.type === DIR_TYPE
 
 /**
+ * Is file param a correct note
  *
  * @param {IOCozyFile} file io.cozy.files
+ * @returns {boolean}
  */
 export const isNote = file => {
-  if (
+  return !!(
```

Comment:
Since I'm still the mainteneur, I'll not accept this change. It brings to much overhead in my brain to understand the double negation. I find that it brings a lot of complexity to understand compared to a simple if/else. Sorry.